x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -88,6 +93,19 @@ const resultToOption = (result: GlobalSearchResult): EuiSelectableTemplateSitewi
     option.meta = [{ text: cleanMeta(type) }];
   }
 
+  if (getTag && tags.length) {
+    const tagBadges = tags.map((tag) => {
+      const { color, name, id: tagId } = getTag(tag)!;
+      return (
+        <EuiBadge color={color} key={tagId}>
+          {name}
+        </EuiBadge>
+      );
+    });
+
+    option.append = <ul style={{ display: 'inline' }}>{tagBadges}</ul>;
```

Comment:
You should just use `SavedObjectsTaggingApi.ui.components.TagList` instead. 
1. this would avoid having to add the `getTag` API as the component is already connected
2. it would allow to adapt code in a single place when we'll add options such as `exceptMaxLength` or `maxDisplayedTags` to enhance the tag list component to address the 'tag list is overflowing' issue

Reply:
Because results aren't saved objects and because of the impending feature freeze, going to punt on this.

Opened an issue to circle back on this: https://github.com/elastic/kibana/issues/85189